src/mail-app/mail/view/MailView.ts

```diff
@@ -585,13 +585,14 @@ export class MailView extends BaseTopLevelView implements TopLevelView<MailViewA
 	 *Shortcut Method to show Labels dropdown only when atleast one mail is selected.
 	 */
 	private labels() {
-		const mailList = this.mailViewModel.listModel
-		if (mailList == null) {
+		const mailList = assertNotNull(this.mailViewModel.listModel)
```

Comment:
we should not unwrap it I think, we might not have a list model if the connection is too slow. Before it was checking for it and we should keep it this way.